internal/models/iam_policy/policy_subject_kind.go

```diff
@@ -36,8 +36,8 @@ const (
 	// VmwareTanzuCoreV1alpha1PolicySubjectKindGROUP captures enum value "GROUP".
 	VmwareTanzuCoreV1alpha1PolicySubjectKindGROUP VmwareTanzuCoreV1alpha1PolicySubjectKind = "GROUP"
 
-	// VmwareTanzuCoreV1alpha1PolicySubjectKindSERVICEACCOUNT captures enum value "SERVICEACCOUNT".
-	VmwareTanzuCoreV1alpha1PolicySubjectKindSERVICEACCOUNT VmwareTanzuCoreV1alpha1PolicySubjectKind = "SERVICEACCOUNT"
```

Comment:
Let's not remove `SERVICEACCOUNT` as it is supported in the API schema

Reply:
added this back